INSTALL.md

```diff
@@ -32,6 +35,13 @@ Prerequisites for building documentation:
 Please see the section "Prerequisites for Optional Features" below additional
 optional prerequisites.
 
+Install prerequisites on Debian GNU/Linux 'Stretch' 9:
+
+    apt-get install gcc cmake libglib2.0-dev libgnutls28-dev libpq-dev postgresql-server-dev-9.6 pkg-config libical-dev
+
+Optional prerequisites on Debian GNU/Linux 'Stretch' 9:
```

Comment:
This is confusing because there's another section for optional features below.

I think it would be clearer if there was an apt command below each case above.  So, one
command below "Prerequisites for certificate generation", one below
"Prerequisites for S/MIME suport (e.g. email encryption)" etc.

Reply:
Indeed. It could make sense to raise an issue at https://github.com/greenbone/templates/issues to have one single base how such an instruction should look like.

Basically every component currently has different install instructions when it comes to prerequisites, quite often they are also just missing mandatory components like libical-dev or gpgsm.

Reply:
Raised https://github.com/greenbone/templates/issues/17. Suggestions on how this should look like are welcome.

Reply:
How to install the dependencies on debian is quiet special and might be better in an additional file. Figuring that out is a task for people packaging the software for the distros. In best case, a user should only install a package and everything is working as expected. We can link to community maintained packages. See for example: https://repology.org/project/gvmd/versions

The INSTALL.md contains also many informations that belongs in the Usage section or external documentation, but that's a different topic.

Reply:
@davidak I have cross-posted this in https://github.com/greenbone/templates/issues/17, its probably the best to handle this in a more broader scope because this is not only affecting gvmd.